aries-site/src/pages/components/checkboxgroup.js

```diff
@@ -0,0 +1,160 @@
+import React from 'react';
+
+import { Anchor, Text } from 'grommet';
+import {
+  CardGrid,
+  Meta,
+  Status,
+  SubsectionText,
+  BulletedList,
+} from '../../components';
+import { ContentSection, Layout, Subsection, Example } from '../../layouts';
+import {
+  CheckBoxGroupDisabledExample,
+  CheckBoxGroupDescriptionExample,
+  CheckBoxGroupSimpleExample,
+  CheckBoxGroupValidationExample,
+} from '../../examples/components/checkboxgroup';
+import { getPageDetails, getRelatedContent, nameToPath } from '../../utils';
+
+const title = 'CheckBoxGroup';
+const topic = 'Components';
+const page = getPageDetails(title);
+const relatedContent = getRelatedContent(title);
+
+const CheckBoxGroup = () => (
+  <Layout title={title}>
+    <Meta
+      title={title}
+      description={page.seoDescription}
+      canonicalUrl="https://design-system.hpe.design/components/checkboxgroup"
+    />
+    <ContentSection>
+      <Subsection name={title} level={1} topic={topic}>
+        <SubsectionText>{page.description}</SubsectionText>
+        {page.status && <Status status={page.status} />}
+        <Example
+          //   docs="https://v2.grommet.io/checkboxgroup?theme=hpe#props"
```

Comment:
Has an issue been created to remind us to add Docs link once available on Grommet?

Reply:
I'll add an issue.

Reply:
Done.